src/js/index.js

```diff
@@ -45,21 +44,21 @@ export {
   MediaChromeButton,
   MediaContainer,
   MediaController,
-  MediaChromeHTMLElement,
   MediaChromeRange,
   MediaControlBar,
   MediaCurrentTimeDisplay,
   MediaDurationDisplay,
+  MediaTimeDisplay,
   MediaSeekForwardButton,
   MediaFullscreenButton,
   MediaMuteButton,
   MediaPipButton,
   MediaPlayButton,
   MediaPlaybackRateButton,
-  MediaPoster,
   MediaProgressRange,
   MediaSeekBackwardButton,
-  MediaThumbnailPreviewElement,
+  MediaThumbnailPreview,
+  MediaThumbnailPreview as MediaThumbnailPreviewElement,
```

Comment:
NOTE: Added this for backwards compat but renamed to be consistent with other media ui elements' naming conventions.

Reply:
Cool. I think we might rename this MediaThumbnailImage based on [the newish naming guide](https://github.com/muxinc/media-chrome/blob/main/ARCHITECTURE.md#element-naming).

I'm not really worried about the backwards compat fo this one since it's only been used internally to this point, but fine either way.